chat/basics/ios.md

```diff
@@ -40,22 +33,22 @@ SKYContainer.default().chatExtension?.createDirectConversation(userID: userBen,
 })
 ```
 
-This example shows how to create a direct chat between the current user and `userBen`.
+The above example creates a direct chat between the current user and `userBen`.
 
-You can also set up optional `metadata` in your conversation for custom configurations.
+You can also set up an optional parameter, `metadata`, for customization.
```

Comment:
Better give a sample? e.g.
"You can also set up an optional parameter, `metadata`, for customisation such as chat title or description."